Program.cs

```diff
@@ -2,11 +2,16 @@
 Console.Write ("Enter a word or phrase:");
 string text = Console.ReadLine ();
 text = text.Replace (" ", "").ToLower ();
+if (text.Equals (ReverseString (text))) Console.WriteLine (text + " is a Palindrome!");

```

Comment:
Still the output is prints like "steponnopets is a Palindrome!".
The output will prints like "Step on no pets".